packages/odyssey-react-mui/src/ui-shell/SideNav/SideNavItemContent.tsx

```diff
@@ -104,6 +107,10 @@ export const getBaseNavItemContentStyles = ({
   borderRadius: odysseyDesignTokens.BorderRadiusMain,
   transition: `backgroundColor ${odysseyDesignTokens.TransitionDurationMain}, color ${odysseyDesignTokens.TransitionDurationMain}`,
   cursor: "pointer",
+  backgroundColor: isDraggedOver
```

Comment:
this is needed to highlight the sidenav item when an app card is dragged over it.